tests/wrappers/test_record_video.py

```diff
@@ -29,6 +31,36 @@ def test_record_video_using_default_trigger():
     shutil.rmtree("videos")
 
 
+def test_record_video_reset_return_info():
+    env = gym.make("CartPole-v1")
+    env._max_episode_steps = 20
```

Comment:
I might be missing something, does this line actually do anything? Directly accessing a `_parameter` is a bit iffy

Reply:
This was just due to a naive copy of another test being used as a template.  It is now removed.